include/wx/image.h

```diff
@@ -598,6 +602,9 @@ class WXDLLIMPEXP_CORE wxImage: public wxObject
 protected:
     static wxList   sm_handlers;
 
+    // choose default allocator for pixel buffers.
+    static wxMemoryAllocator *sm_alloc;
```

Comment:
Code Guidelines say that member static variables should use `ms_` prefix. However, it could be odd as the variable declared just above, does not abide the rule...